discounts-react-app/src/broken-files/src/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+import { datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+  applicationId: `${
+    import.meta.env.REACT_APP_DD_APPLICATION_ID ||
+    'DD_APPLICATION_ID_PLACEHOLDER'
+  }`,
+  clientToken: `${
+    import.meta.env.REACT_APP_DD_CLIENT_TOKEN || 'DD_CLIENT_TOKEN_PLACEHOLDER'
+  }`,
+  site: `${import.meta.env.REACT_APP_DD_SITE || 'datadoghq.com'}`,
+  service: `${import.meta.env.REACT_APP_DD_SERVICE || 'discounts-frontend'}`,
+  version: `${import.meta.env.REACT_APP_DD_VERSION || '1.0.0'}`,
+  env: `${import.meta.env.REACT_APP_DD_ENV || 'production'}`,
+  sampleRate: 100,
+  trackInteractions: true,
+  defaultPrivacyLevel: 'mask-user-input',
+  allowedTracingOrigins: [
+    /https:\/\/.*\.environments.katacoda\.com/,
```

Comment:
Can you update it to this so it matches the new Instruqt path?

```suggestion
    /https:\/\/.*\.env.play.instruqt\.com/,
```